utils/data_archive/archive_explorer.py

```diff
@@ -101,3 +101,21 @@ def get_most_recent_run_since(self, instrument, cut_off_time):
         if valid_files:
             return valid_files[-1]
         return None
+
```

Comment:
Changes to archive explorer can be removed from this PR as it is no longer required due to querying ICAT